app/controllers/pages_controller.rb

```diff
@@ -8,6 +8,10 @@ def about
     @title = "About Us"
   end
 
+  def calendar
+    @title = "Calendar"
```

Comment:
Doesn't look like you use this variable - so we can get rid of. If you're going to set it to a non-dynamic string, we can do that in the view anyway.